### PR TITLE
Change tabs with modifier+digit

### PR DESCRIPTION
**Pull request by nb**
I have a very strong muscle memory for changing tabs via `⌘+1`, `⌘+2`,
etc.

Even though it’s a little strange in the FIFO tabs model I got used to
it pretty quickly.

### CODE REVIEW

**Comment by spolu** [comment]
That's awesome. Can you rename the event to `select_by_index` and remove the `_tab_` from the method name in `strip.js` ?

**Comment by nb** [comment]
Sure, those names make more sense.

Just force-pushed.

File: lib/strip.js [hunk 1]
```diff
@@ -381,6 +382,17 @@ var strip = function(spec, my) {
     }
   };
 
+  // ### shortcut_select_by_index
+  //
+  // Keyboard shortcut to select a specific tab, based on its index, not on the tab id
+  shortcut_select_by_index = function(tab_index) {
```
Comment (spolu) [inline lib/strip.js:391]: sorry last thing on the structure of the comment. It should specify the arguement:

``````
// ### shortcut_select_by_index
//
// Keyboard shortcut to select a specific tab, based on its index, not on the tab id
// ```
// @tab_index {number} the tab index [0-9]
// ```
``````

Comment (nb) [inline lib/strip.js:391]: Of course. Fixed.

**Comment by spolu** [comment]
sweet!

File: lib/strip.js (hunk 1 above)
Comment (nb) [inline lib/strip.js:391]: By the way, are you extracting those doc-strings somehow? If not, rehashing what the function does is tedious and error-prone. The chance of me not updating the comment after changing the function is pretty high.

Comment (spolu) [inline lib/strip.js:391]: That's true. Do you think it's better to remove the arguments altogether?
No extracting them for now.

Comment (nb) [inline lib/strip.js:391]: In my experience the doc blocks are rarely useful and their meaning is better conveyed with good names (just what you pushed me to do) or with simpler functions.

Comment (spolu) [inline lib/strip.js:391]: I find they help readiblity of the code even if noone reads their content :)

Comment (nb) [inline lib/strip.js:391]: I am curious, why? Is it the visual separation? Or the feeling of “documentedness”?

Comment (spolu) [inline lib/strip.js:391]: Probably both. But I find the visual separation quite important...

Comment (nb) [inline lib/strip.js:391]: Fair enough. I don’t agree, but this is a great project, your project, and we can spend our time in more productive ways.

Comment (spolu) [inline lib/strip.js:391]: Thanks ;-)

**Comment by spolu** [comment]
published v0.2.9-alpha.5 should get to you through auto-update

**Pull request merged**